exchanges/kraken/kraken_wrapper.go

```diff
@@ -1263,6 +1268,7 @@ func (k *Kraken) GetActiveOrders(ctx context.Context, req *order.MultiOrderReque
 				Side:            side,
 				Type:            orderType,
 				Pair:            p,
+				AssetType:       asset.Spot,
```

Comment:
Something I noticed but doesn't stop merge. 

Status is `Unknown` raw response has `open`

Raw response: 

```
{"error":[],"result":{"open":{"ODGFZI-E4GA6-VTPRDR":{"refid":null,"userref":0,"status":"open","opentm":1690768649.226174,"starttm":0,"expiretm":0,"descr":{"pair":"XBTUSDT","type":"buy","ordertype":"limit","price":"10000.0","price2":"0","leverage":"none","order":"buy 0.00010000 XBTUSDT @ limit 10000.0","close":""},"vol":"0.00010000","vol_exec":"0.00000000","cost":"0.00000","fee":"0.00000","price":"0.00000","stopprice":"0.00000","limitprice":"0.00000","misc":"","oflags":"fciq"}}}}
```
type:

```
[{ImmediateOrCancel:false HiddenOrder:false FillOrKill:false PostOnly:false ReduceOnly:false Leverage:0 Price:10000 Amount:0.0001 LimitPriceUpper:0 LimitPriceLower:0 TriggerPrice:0 AverageExecutedPrice:0 QuoteAmount:0 ExecutedAmount:0 RemainingAmount:0.0001 Cost:0 CostAsset: Fee:0 FeeAsset: Exchange:Kraken InternalOrderID:00000000-0000-0000-0000-000000000000 OrderID:ODGFZI-E4GA6-VTPRDR ClientOrderID: AccountID: ClientID: WalletAddress: Type:LIMIT Side:BUY Status:UNKNOWN AssetType:spot Date:2023-07-31 11:57:29.226174116 +1000 AEST CloseTime:0001-01-01 00:00:00 +0000 UTC LastUpdated:0001-01-01 00:00:00 +0000 UTC Pair:XBT_USDT Trades:[]}]
```